src/zenml/cli/model.py

```diff
@@ -559,57 +538,43 @@ def _print_artifacts_links_generic(
         f"{type_} linked to the model version `{model_version.name}[{model_version.number}]`:"
     )
 
-    model_version = Client().get_model_version(
-        model_name_or_id=model_name_or_id,
-        model_version_name_or_number_or_id=model_version_name_or_number_or_id,
-    )
-
     links = Client().list_model_version_artifact_links(
         model_version_id=model_version.id,
-        model_version_artifact_link_filter_model=ModelVersionArtifactFilterModel(
-            only_data_artifacts=only_data_artifacts,
-            only_endpoint_artifacts=only_endpoint_artifacts,
-            only_model_artifacts=only_model_artifacts,
-            **kwargs,
-        ),
+        only_data_artifacts=only_data_artifacts,
+        only_endpoint_artifacts=only_endpoint_artifacts,
+        only_model_artifacts=only_model_artifacts,
+        **kwargs,
     )
 
     cli_utils.print_pydantic_models(
         links,
-        columns=[
-            "pipeline_name",
-            "step_name",
-            "name",
-            "link_version",
-            "artifact",
-            "created",
-        ],
+        columns=["artifact", "created"],
     )
 
 
 @version.command(
     "data_artifacts",
     help="List data artifacts linked to a model version.",
 )
-@click.argument("model_name_or_id")
-@click.argument("model_version_name_or_number_or_id", default="0")
```

Comment:
Don't like this change in the way it is done now. Can we move this up to model group? Version group assume that version is an argument in all methods.

Reply:
It was already an optional argument before so conceptually nothing really changed

Reply:
but sure we can move it to model

Reply:
I'm just upvoting consistency here, if I call `zenml model version` any command, I expect to be able to pass `MODEL_NAME MODEL_VERSION` as unnamed params. This is the only concern. Rest is all good.

Reply:
We can also do it in subsequent PR, tbh.

Reply:
Already done 👍